docs/getting_started/tutorials.md

```diff
@@ -2,6 +2,10 @@
 
 This is a collection of `LangChain` tutorials on `YouTube`.
 
+⛓ icon marks a new tutorial
```

Comment:
should we add an update date here
```x icon marks a new tutorial, last updated 05.15.2023```

Reply:
> should we add an update date here `x icon marks a new tutorial, last updated 05.15.2023`

Not sure about it. Plus, it is sooo much additional work.

I'm going to update this icon on a weekly or monthly basis. Remove it from the previous update and add it to new links. 

The user is interested in the "new" status not in the "when exactly it was added" date, IMO. If so, the icon is better than the exact update date.

Reply:
Ohhhh, you don't ask about date for each video, but date new a legend for this icon! Yes, sure. Good idea.